docs/get-started.md

```diff
@@ -46,7 +48,47 @@ cd -
 gittuf version
 ```
 
-**Building from source.** To build from source, clone the repository and run
+### Windows
+
+Run this script as a `.ps1` file (PowerShell script).
```

Comment:
Sure, we can add that blurb in.

> On the other hand, having the user make a .bat file with the commands may prove to be simpler.

Instead of having the user make a .bat file, wouldn't it be better if we just provide one that does the same thing? And all the user has to do is download and run it